src/status_im2/contexts/scan/view.cljs

```diff
@@ -0,0 +1,44 @@
+(ns status-im2.contexts.scan.view
```

Comment:
This one is more for the CC team. There has been much discussion about what is in each context. Anyway for this particular instance I think we should move all of the elements of this figma file into one context ("Shell"). 
i.e this design file: https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?type=design&node-id=2545-165763&mode=design&t=NOYKKKdFVBgDZlic-0

Otherwise the contexts will just get too large and it will be somewhat pointless.

The small change from current structure I am proposing is this:

```
src/status-im2/contexts/shell
|-activity-center/....
|-jump-to/....  (I think we currently have this as shell, we should rename it as shell encompasses all of these things)
|-scan/....   (the ns used in this pr.)
|-share/....
```

@flexsurfer, @Parveshdhull, @ilmotta, @siddarthkay thoughts with putting all of these in the same context?

Reply:
Seems fair to me..

Reply:
convert this discussion to an issue ?

Reply:
The name `contexts` comes from `bounded contexts` from DDD, which imply we want to use an [ubiquitous language](https://www.martinfowler.com/bliki/UbiquitousLanguage.html). So IMO the contexts should try hard to respect this idea.

I think your suggestion @J-Son89 for using `shell` is the correct one because it's how the Design team and dev teams refer to them.

Reply:
https://github.com/status-im/status-mobile/issues/16360